mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -102,13 +90,70 @@ export class ExperimentListView extends Component {
     this.updateSelectedExperiment('0', '');
   };
 
+  onSelect = (experimentId) => () => {
+    const { history, activeExperimentId } = this.props;
+    if (experimentId === activeExperimentId) {
+      return;
+    }
+    history.push(Routes.getExperimentPageRoute(experimentId));
+  };
+
+  renderListItem = ({ title, key }) => {
+    const { activeExperimentId } = this.props;
+    const dataTestId =
+      activeExperimentId === key ? 'active-experiment-list-item' : 'experiment-list-item';
+    return (
+      <div style={{ display: 'flex', marginLeft: '8px' }} data-test-id={dataTestId}>
+        <div
+          style={{
+            width: '180px',
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            whiteSpace: 'nowrap',
+          }}
+          onClick={this.onSelect(key)}
+        >
+          {title}
+        </div>
```

Comment:
Is there a reason we don't use LinkTo here and rather use `div`? This isn't great for a11y as well since the screen readers wouldn't read it as Link and note that there would a transition. Plus tracking things like site-speed is difficult where we rely on navigation or something.

Reply:
@sunishsheth2009

> Is there a reason we don't use LinkTo here and rather use div?

No, we can replace this div with `<Link to=... >`.